### PR TITLE
Announce Request pages

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 66a11f231e980537c3586287f13f5edbe1e71829

https://deploy-preview-216--openbuildservice.netlify.com

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 9c15f20813196ee11dd5a538218d77ed2b611191

https://deploy-preview-216--openbuildservice.netlify.com

**Comment by dmarcoux** [comment]
Removed duplicated bot message.

**Comment by hennevogel** [comment]
> Removed duplicated bot message.

Whom are you calling a bot? ;-) Chefbot!

chefbot

**Comment by dmarcoux** [comment]
[Communicate the deliverable to the customers in a proper way](https://trello.com/c/uMBgi297/1024-communicate-the-deliverable-to-the-customers-in-a-proper-way)

**Comment by DavidKang** [comment]
@dmarcoux, nice! LGTM

**Comment by bgeuken** [comment]
If it's still possible, I would add a spec file and a changes file in the first gif

File: _posts/development/2019-02-15-revamping-request-ui.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+We refreshed the page while keeping its current form.
+We added some quality of life improvements, such as the `Expand/Collapse all` buttons for the diffs.
+
+<img src="/images/posts/revamping-request-ui/expand-collapse-all.gif" alt="Expand/Collapse all buttons" style="width: 80%;" class="center" />
+
+The comments from a superseded request are now under a tab, next to the request's comments.
+
+<img src="/images/posts/revamping-request-ui/superseded-comments.gif" alt="Superseded comments" style="width: 80%;" class="center" />
+
+## Improvements for the Project Monitor Page
```
Comment (saraycp) [inline _posts/development/2019-02-15-revamping-request-ui.md]: Thanks for taking care again :heart:  Only one [nitpick]:
This section looks a bit empty and there is related text in section _And Much More_ (line 53)

File: _posts/development/2019-02-15-revamping-request-ui.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+New year, new milestone!
+We have migrated the request pages to the new user interface and addressed issues regarding the previous releases (thanks to your feedback!).
+These changes are already available for all users in the [beta program](/2018/10/04/the-beta-program).
+This is following up on our [previous announcement](/2018/12/10/revamping-project-ui) from December 2018,
+where we presented the revamped user interface for Project, Package, User and Group pages.
+
+Here is a comparison of the old and new user interface of the request overview page:
+
+<img src="/images/posts/revamping-request-ui/comparison-old-new.gif" alt="Comparison of the old and new user interface" style="width: 80%;" class="center" />
+
```
Comment (hennevogel) [inline _posts/development/2019-02-15-revamping-request-ui.md]: I don't like that it looks like we dropped the top bar...

File: _posts/development/2019-02-15-revamping-request-ui.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Another Milestone for the Revamped User Interface, Now for the Request Pages
+category: development
+---
+
+New year, new milestone!
+We have migrated the request pages to the new user interface and addressed issues regarding the previous releases (thanks to your feedback!).
+These changes are already available for all users in the [beta program](/2018/10/04/the-beta-program).
+This is following up on our [previous announcement](/2018/12/10/revamping-project-ui) from December 2018,
```
Comment (hennevogel) [inline _posts/development/2019-02-15-revamping-request-ui.md]: I think you should collect the links to previous announcements in a partial and include it in all posts. Something like:

This release is part of a series of posts about OBS moving to a new UI. We started off with the [Package pages](/2018/10/05/revamping-ui/) in October 2018, moved on to the [Project, User and Group pages](/2018/12/10/revamping-project-ui) and just now finished the [Request pages](/2019/02/15/revamping-request-ui/).

If you put this in to an jekyll include file and include it from all posts then all posts always link to each other.

File: _posts/development/2019-02-15-revamping-request-ui.md (hunk 1 above)
Comment (dmarcoux) [reply _posts/development/2019-02-15-revamping-request-ui.md]: @Ana06 should be writing here. If we don't have it by tomorrow, I will simply remove it before merging.

File: _posts/development/2019-02-15-revamping-request-ui.md (hunk 3 above)
Comment (dmarcoux) [reply _posts/development/2019-02-15-revamping-request-ui.md]: Great idea!

I've done it, so please have a look.

File: _posts/development/2019-02-15-revamping-request-ui.md (hunk 2 above)
Comment (dmarcoux) [reply _posts/development/2019-02-15-revamping-request-ui.md]: Fixed, even though it's a bit difficult to have a GIF with a square ratio (so it looks better).

File: _posts/development/2019-02-15-revamping-request-ui.md (hunk 1 above)
Comment (dmarcoux) [reply _posts/development/2019-02-15-revamping-request-ui.md]: I wrote something without the performance.

**Comment by dmarcoux** [comment]
I'm done, feel free to review and merge if you're happy with the result.

**Pull request merged**